From 7a04a41598498ebb4ed1f1964401604fa33fe89d Mon Sep 17 00:00:00 2001
From: Julian Greilich <j.greilich@gmx.de>
Date: Mon, 18 May 2026 15:22:41 +0200
Subject: a11y Windows: Enable ExpandCollapsePattern for all expandable
 elements

All elements that are expandable should be announced as that.

Task-number: QTBUG-146552
Change-Id: Ia658cd3b19a0655ee2b2deb18dca3abc1a1f5b82
---
 .../platforms/windows/uiautomation/qwindowsuiamainprovider.cpp  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git x/qtbase/src/plugins/platforms/windows/uiautomation/qwindowsuiamainprovider.cpp y/qtbase/src/plugins/platforms/windows/uiautomation/qwindowsuiamainprovider.cpp
index 92c3997922c5d9407e74af22382d4ceced187143..2e0d99958e1c7858b3407600bce51c41b4f20428 100644
--- x/qtbase/src/plugins/platforms/windows/uiautomation/qwindowsuiamainprovider.cpp
+++ y/qtbase/src/plugins/platforms/windows/uiautomation/qwindowsuiamainprovider.cpp
@@ -373,7 +373,7 @@ HRESULT QWindowsUiaMainProvider::GetPatternProvider(PATTERNID idPattern, IUnknow
                 && accessible->childCount() > 0
                 && accessible->child(0)->role() == QAccessible::PopupMenu)
             || accessible->role() == QAccessible::ComboBox
-            || (accessible->role() == QAccessible::TreeItem && accessible->state().expandable)) {
+            || accessible->state().expandable) {
             *pRetVal = makeComObject<QWindowsUiaExpandCollapseProvider>(id()).Detach();
         }
         break;
